bindata/v4.1.0/kube-controller-manager/pod.yaml

```diff
@@ -10,27 +10,17 @@ metadata:
     kube-controller-manager: "true"
     revision: "REVISION"
 spec:
-  initContainers:
-  - name: wait-for-host-ports
-    terminationMessagePolicy: FallbackToLogsOnError
-    image: ${IMAGE}
-    imagePullPolicy: IfNotPresent
-    command: ['/usr/bin/timeout', '65', '/bin/bash', '-ec'] # a bit more than 60s for TIME_WAIT, 5s extra cri-o's graceful termination period
-    args:
-    - |
-      echo -n "Waiting for port :10257 and :10357 to be released."
-      while [ -n "$(lsof -ni :10257)$(lsof -ni :10357)" ]; do
-        echo -n "."
-        sleep 1
-      done
   containers:
   - name: kube-controller-manager
     image: ${IMAGE}
     imagePullPolicy: IfNotPresent
     terminationMessagePolicy: FallbackToLogsOnError
-    command: ["/bin/bash", "-ec"]
+    command: ["/bin/bash", "-euxo", "pipefail", "-c"]
     args:
         - |
+          echo -n "Waiting for port :10257 to be released."
+          timeout 3m /bin/bash -euo pipefail -c 'while [ -n "$(lsof -ni :10257)" ]; do echo -n "."; sleep 1; done'
```

Comment:
please don't diverge from https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/bindata/v4.1.0/kube-apiserver/pod.yaml. This lsof does not work.

Reply:
I have not noticed you changed it since then. What didn't work about the lsof? I have tried it locally but sounds like there is corner case?

Reply:
Yeah, I had https://github.com/openshift/cluster-kube-controller-manager-operator/pull/418/ changing it but it didn't merge. @tnozicka can you pick that up here.

Reply:
converted to `ss`